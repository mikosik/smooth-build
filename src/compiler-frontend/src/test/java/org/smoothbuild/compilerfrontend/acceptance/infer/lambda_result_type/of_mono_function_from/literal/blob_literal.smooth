
myValue = () -> 0x07;