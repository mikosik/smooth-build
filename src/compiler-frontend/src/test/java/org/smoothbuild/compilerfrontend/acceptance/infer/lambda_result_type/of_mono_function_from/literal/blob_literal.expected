()->Blob myValue<>
  = <>() -> 0x07<>;
